### PR TITLE
Cavonac/sqlassessment

### PR DESCRIPTION
**Pull request by cavonac**
This PR updates the SQL-Assessment notebook for the Hybrid toolkit. It has been tested, updated for content and clarity. Removed registered servers references because it was confusing to the notebook task. We can place another notebook for managing and migrating registered servers later if needed. For now, retrieving the assessment was left as the focus. A new sql command was added to display the results of the assessment in the notebook for the user.

### CODE REVIEW

File: extensions/azurehybridtoolkit/notebooks/hybridbook/content/Assessments/sql-server-assessment.ipynb [hunk 1]
```diff
@@ -49,12 +50,12 @@
...
             "source": [
-                "## Notebook Steps\r\n",
-                "1.  Ensure that the proper APIs and modules are installed per the <a href=\"../prereqs.ipynb\">prerequisites</a> notebook\r\n",
-                "2.  Define a service instance and group corresponding to the SQL Server instances to be assessed\r\n",
-                "3.  Choose an example below that corresponds to the appropriate task\r\n",
-                "4.  Execute only that example's code block and wait for results\r\n",
-                "5.  Fix any recommended issues and rerun Assessment API until clear"
+                "## Notebook Steps\n",
+                "\n",
+                "1. Ensure that the proper APIs and modules are installed per the [prerequisites](.\\.................\\prereqs.ipynb) notebook\n",
```
Comment (Charles-Gagnon) [inline extensions/azurehybridtoolkit/notebooks/hybridbook/content/Assessments/sql-server-assessment.ipynb:55]: Are the extra .'s here intended?

**Comment by Charles-Gagnon** [comment]
@smartguest FYI (for some reason I can't add you as a reviewer...)

File: extensions/azurehybridtoolkit/notebooks/hybridbook/content/Assessments/sql-server-assessment.ipynb (hunk 1 above)
Comment (cavonac) [reply extensions/azurehybridtoolkit/notebooks/hybridbook/content/Assessments/sql-server-assessment.ipynb:55]: No, there is a bug in ADS that replaces links I have when I move from markdown. This should be a relative link to the prereqs notebook, not a string of periods. should just be two at most to go "up" a folder.

Comment (Charles-Gagnon) [reply extensions/azurehybridtoolkit/notebooks/hybridbook/content/Assessments/sql-server-assessment.ipynb]: Ah interesting - if you have a repro for that could you file an issue for it? That definitely sounds like a problem!

@chlafreniere @lucyzhang929 @barbaravaldez

**Review by smartguest — APPROVED** [review]

**Pull request merged**